### PR TITLE
Add Clausweb to hosting providers

### PR DESCRIPTION
**Pull request by Vudubond**

### CODE REVIEW

File: _data/hosting_providers.json [hunk 1]
```diff
@@ -2509,4 +2509,14 @@
     "reviewed": "2021.02.12",
     "note": "All webhosting plans include Let's Encrypt."
   }
```
Comment (ohemorange) [inline _data/hosting_providers.json:2511]:
```suggestion
  },
```

File: _data/hosting_providers.json [hunk 2]
```diff
@@ -2509,4 +2509,14 @@
...
+   {
```
Comment (ohemorange) [inline _data/hosting_providers.json:2512]:
```suggestion
  {
```

File: _data/hosting_providers.json [hunk 3]
```diff
@@ -2509,4 +2509,14 @@
...
+    "name": "Clausweb",
+    "link": "https://www.clausweb.ro/",
+    "category": "full",
+    "tutorial": "",
+    "announcement": "https://www.clausweb.ro/blog/ce-este-un-certificat-ssl-si-la-ce-se-foloseste/",
+    "plan": "",
+    "reviewed": "2019.5.31",
```
Comment (ohemorange) [inline _data/hosting_providers.json:2519]:
```suggestion
    "reviewed": "2020.2.19",
```

**Review by ohemorange — CHANGES_REQUESTED** [review]
This is great, thanks for supporting encryption on the web! I have a couple minor fixes, but otherwise looks good to go.

**Comment by Vudubond** [comment]
Changes have been made.

**Review by Vudubond — COMMENTED** [review]
Changes made.

**Review by ohemorange — APPROVED** [review]
looks great, thank you!

**Pull request merged**